main/python-bcrypt/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "python-bcrypt"
+pkgver = "4.2.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-setuptools-rust",
+    "python-cffi",
+    "python-build",
+    "python-installer",
+    "cargo",
+]
+makedepends = ["python-devel", "rust-std"]
+depends = ["python-cffi"]
```

Comment:
i don't see cffi used anywhere in the codebase, or for build, or for tests, and they pass without it. are you sure it's used?

Reply:
I really just shouldn't look at how others package something. cffi was used *until* version 4, which is when python-bcrypt was rewritten in rust.

Reply:
(Thanks for catching this.)